pkg/kubectl/scale_test.go

```diff
@@ -63,8 +65,8 @@ func (c *ErrorReplicationControllerClient) ReplicationControllers(namespace stri
 }
 
 func TestReplicationControllerScaleRetry(t *testing.T) {
-	fake := &ErrorReplicationControllerClient{Clientset: fake.NewSimpleClientset(oldRc(0, 0)), conflict: true}
-	scaler := ReplicationControllerScaler{fake}
+	faker := &ErrorReplicationControllerClient{Clientset: fake.NewSimpleClientset(oldRc(0, 0)), conflict: true}
```

Comment:
no need to change the name here, please revert to `fake` so that we can minimize difs.